src/test/run-make/llvm-module-pass/Makefile

```diff
@@ -0,0 +1,9 @@
+-include ../tools.mk
+
+all:
+	$(CXX) $(LLVM_CXXFLAGS) -c llvm-pass.so.cc -o $(TMPDIR)/llvm-pass.o
+	$(AR) rcs $(TMPDIR)/libllvm-pass.a $(TMPDIR)/llvm-pass.o
```

Comment:
Can this use the support in `tools.mk`? The `all` target can depend on `$(call NATIVE_STATICLIB,llvm-pass)` and then you can define a rule for `$(TMPDIR)/llvm-pass.o` which depends on `llvm-pass.so.cc`.